### DISCUSSION TITLE
JWT with docker/containerd cli clients

### DISCUSSION THREAD

**Discussion by amitkumar2283** [distribution/distribution#4369] [General]
We are able to generate JWT token per specifications. We can also use the JWT token to access v2 APIs.

But we are not able to use docker-cli or crictl to pull images using JWT.
I have tried:
`command:`

````
docker login -u jwt -p <generatedJWTtoken> <harbor-hostname>
````

`output`

````
Error response from daemon: Get "https://<registry>/v2/": unauthorized:
````

Please suggest how to use JWTs with CLI

**Comment by milosgajdos** [comment]
`docker login` does NOT accept JWTs, so what you are trying to do will not work.

`docker login --help` 👇 

> Options:
>  -p, --password string   Password
